### PR TITLE
how-to-send-a-signal-from-a-workflow-in-python

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
Adds `how-to-send-a-signal-from-a-workflow-in-python`.

Code sample:

```python
@workflow.defn
class MyWorkflow:
    @workflow.run
    async run(self) -> None:
        handle = workflow.get_external_workflow_handle_for(OtherWorkflow.run, "other-workflow-id")
        await handle.signal(OtherWorkflow.other_signal, "other signal arg")
```

### CODE REVIEW

File: docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+id: how-to-send-a-signal-from-a-workflow-in-python
+title: How to Send Signal from Workflow in Python
```
Comment (djmagee) [inline docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md:3]:
```suggestion
title: How to Send a Signal from a Workflow in Python
```

File: docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_label: Send Signal from Workflow
+description: Send Signal from Workflow.
```
Comment (djmagee) [inline docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md:5]:
```suggestion
description: Use `get_external_workflow_handle_for` to get a typed Workflow handle to an existing Workflow by its identifier.
```

File: docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+  - developer-guide
+  - sdk
+  - python
+---
+
+Use the [`get_external_workflow_handle_for`](https://python.temporal.io/temporalio.workflow.html#get_external_workflow_handle_for) to get a typed Workflow handle to an existing Workflow by its identifier.
```
Comment (djmagee) [inline docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md:12]:
```suggestion
Use [`get_external_workflow_handle_for`](https://python.temporal.io/temporalio.workflow.html#get_external_workflow_handle_for) to get a typed Workflow handle to an existing Workflow by its identifier.
```

File: docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+title: How to Send a Signal from a Workflow in Python
+sidebar_label: Send Signal from Workflow
+description: Use `get_external_workflow_handle_for` to get a typed Workflow handle to an existing Workflow by its identifier.
+tags:
+  - developer-guide
+  - sdk
+  - python
+---
+
+Use [`get_external_workflow_handle_for`](https://python.temporal.io/temporalio.workflow.html#get_external_workflow_handle_for) to get a typed Workflow handle to an existing Workflow by its identifier.
```
Comment (cretz) [inline docs-src/python/how-to-send-a-signal-from-a-workflow-in-python.md:12]: Note that `get_external_workflow_handle` is used when you don't know the type of the other workflow. These mimic the `client.get_workflow_handle` and `client.get_workflow_handle_for` methods.

**Review by cretz — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Pull request merged**